redaxo/src/core/lib/console/setup/run.php

```diff
@@ -279,22 +279,26 @@ static function ($value) {
 
         $tables_complete = '' == rex_setup_importer::verifyDbSchema();
 
+        $defaultDbMode = ' normal ';
         $createdbOptions = [
             'normal' => 'Setup database',
             'override' => 'Setup database and overwrite it if it exitsts already (Caution - All existing data will be deleted!)',
             'existing' => 'Database already exists (Continue without database import)',
             'update' => 'Update database (Update from previous version)',
             'import' => 'Import existing database export',
         ];
-        if (!$tables_complete) {
+
+        if ($tables_complete) {
+            $defaultDbMode = 'existing';
+        } else {
             unset($createdbOptions['existing']);
         }
         if (0 === count($backups)) {
             unset($createdbOptions['import']);
         }
 
         $createdb = $this->getOptionOrAsk(
-            new ChoiceQuestion('Choose database setup', $createdbOptions, ' normal '),
```

Comment:
@bloep warum hatten wir hier `' normal '` mit spaces vorne/hinten drann?
hatte mich gewundert dass es nicht der gleiche wert ist, wie der hash im array, also `normal` ohne spaces?

Reply:
https://github.com/redaxo/redaxo/pull/3118#issuecomment-578422995 (den hidden comment dort aufklappen).
Es war ein Trick, um in der Anzeige als Default-Wert den Key "normal" zu sehen, statt dem zugehörigen Langtext.

Reply:
danke, hab ich als kommentar ergänzt und auch für "existing" hergenommen:

![grafik](https://user-images.githubusercontent.com/120441/103453605-f19bcb00-4cdb-11eb-84dd-3a006896a747.png)